### PR TITLE
Fix bower deployment if the dist folder already exists

### PR DESCRIPTION
**Pull request by skateman**

### CODE REVIEW

**Comment by himdel** [comment]
.. the idea being that we drop any old `dist/` before replacing it completely with the freshly built one.

And `mv -f tmp dist` will always rename, not move to `dist/tmp` as it did when `dist` already existed.

Waiting for green :+1:

File: ci/bower-dev.sh [hunk 1]
```diff
@@ -7,6 +7,7 @@ mv bower.json bower.json.tmp
 git remote add upstream https://github.com/ManageIQ/ui-components.git
 git fetch upstream
 git checkout bower-dev
+rm -rf dist
```
Comment (Fryguy) [inline ci/bower-dev.sh]: This will delete also any ancillary files like the README and LICENSE, which might have side effects?

Comment (skateman) [inline ci/bower-dev.sh]: README and LICENSE are not in the `dist` folder

Comment (Fryguy) [inline ci/bower-dev.sh]: Oh I see...they are at the root of the repo.  :+1:

**Pull request merged**